src/templates/open-enterprise/index.js

```diff
@@ -0,0 +1,152 @@
+    { appName: 'dot-voting.aragonpm.eth', label: 'Dot Voting' },
+    { appName: 'projects.aragonpm.eth', label: 'Projects' },
+    { appName: 'rewards.aragonpm.eth', label: 'Rewards' },
+  ],
+  screens: [
+    [
+      data => completeDomain(data.domain) || 'Claim domain',
+      props => <ClaimDomainScreen screenProps={props} />,
+    ],
+    ['Configure template', props => <VotingScreen screenProps={props} />],
+    ['Configure template', props => <DotVotingScreen screenProps={props} />],
+    ['Configure template', props => <TokensScreen screenProps={props} />],
+    [
+      'Review information',
+      props => {
+        const { domain, dotVoting, tokens, voting } = props.data
+        return (
+          <ReviewScreen
+            screenProps={props}
+            items={[
+              {
+                label: 'General info',
+                fields: [
+                  ['Organization template', 'Open Enterprise'],
+                  ['Name', completeDomain(domain)],
+                ],
+              },
+              {
+                label: (
+                  <KnownAppBadge appName="voting.aragonpm.eth" label="Voting" />
+                ),
+                fields: VotingScreen.formatReviewFields(voting),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="dot-voting.aragonpm.eth"
+                    label="Dot Voting"
+                  />
+                ),
+                fields: DotVotingScreen.formatReviewFields(dotVoting),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="token-manager.aragonpm.eth"
+                    label="Tokens"
+                  />
+                ),
+                fields: TokensScreen.formatReviewFields(tokens),
+              },
+            ]}
+          />
+        )
+      },
+    ],
+  ],
+  prepareTransactions(createTx, data) {
+    const allocationsPeriod = 0 // default
+    const financePeriod = 0 // default
+
+    const { domain, dotVoting, tokens, voting } = data
+    const { tokenName, tokenSymbol, members } = tokens
+
+    const baseStake = new BN(10).pow(new BN(18))
+    const stakes = members.map(([_, stake]) =>
+      baseStake.mul(new BN(stake.toString())).toString()
+    )
+    const accounts = members.map(([account]) => account)
+
+    const { support, quorum, duration } = voting
+    const onePercent = new BN(10).pow(new BN(16))
+    const adjustedSupport = onePercent.mul(new BN(support)).toString()
+    const adjustedQuorum = onePercent.mul(new BN(quorum)).toString()
+    const adjustedDuration = new BN(duration).toString()
+    const votingSettings = [adjustedSupport, adjustedQuorum, adjustedDuration]
+
+    const {
+      support: dvSupport,
+      quorum: dvQuorum,
+      duration: dvDuration,
+    } = dotVoting
+    const adjustedDvSupport = onePercent.mul(new BN(dvSupport)).toString()
```

Comment:
Does dot voting revert if these percentages are at 100%?

Reply:
It does not revert with 100% on DotVoting, but it does on Voting:
![gif](https://media.giphy.com/media/QC19eE0EDRzpZb5CqO/giphy.gif)

It seems an edge case to use that values, but 🤷‍♂️ hypothetical 100% consensus DAO?

Reply:
Yes, Voting uses strict greater than checks, so choosing 100% means you need 100% + 1 for a vote to pass, which is impossible. That's why we do the "adjustment".